### ISSUE TITLE
Emails are not sending to publisher or editor

### ISSUE THREAD

**Issue by ghost** [engram-design/Workflow#108]
### Description

I have set up some users to be editors and others to be publishers. When an editor creates a post for approval using the front-end form, the publishers don't receive an email notification to say that an entry was created.

### Steps to reproduce

1. Create content on the front-end entry form (not in CP) and submit.
2. Publisher doesn't receive the email.

### Additional info

- Plugin version: 1.4.6
- Craft version: 3.4.17

**Comment by engram-design** [comment]
Can you look at your log file - `storage/logs/workflow.log` and let me know the content's of this? From my tests, emails are being send to the publisher(s) correctly.

**Comment by knynkwl** [comment]
I am also having this issue. We are using SMTP/TLS with postmark.

    Plugin version: 1.5.1
    Craft version: 3.4.22.1

I'm not seeing anythign relevant in the logs:

```
2020-06-08 16:06:50 [-][4779][-][info][workflow] Try to render editor-pane
2020-06-08 16:06:50 [-][4779][-][info][workflow] Rendering editor-pane for #1097769
2020-06-08 16:06:50 [-][4779][-][info][application] $_GET = [
    'p' => 'cms/entries/pages/1091145-highlights-from-virtual-aihce-exp-2020'
    'site' => 'default'
    'draftId' => '5674'
]```

**Comment by engram-design** [comment]
@knynkwl Wondering if this might be related to https://github.com/verbb/workflow/issues/117 - can you check on the latest version?

**Comment by knynkwl** [comment]
@engram-design the client is still not receiving emails. I tried entering my own email address to test and I did not receive an email either. I don't see any logs other than what I added above.

I tested the email settings in craft and it sent fine.

**Comment by knynkwl** [comment]
Also, I did update to 1.5.2 and the issue still persists.

**Comment by knynkwl** [comment]
Also, btw I am not seeing the email being sent through our postmark app, I can see the test email go through from craft and other emails being sent through forms.

**Comment by engram-design** [comment]
Just to confirm, is this being made from a front-end form like the original issue? It might also be a dump question, but I assume the Notifications are enabled for editors and publishers?

Are you also using a `config/workflow.php` file? Or just have all settings set through the control panel? From what I can tell, it seems like the default config settings aren't being respected, causing some issues...

Can you also give [1.5.3](https://github.com/verbb/workflow/releases/tag/1.5.3) a go

**Comment by knynkwl** [comment]
@engram-design Thanks for the update. We are just using the config settings in the control panel.

Here is what we are seeing after the update:

✅ Publishers are now receiving emails. 
❌ Content Editors are not receiving emails that their submission has been approved.

**Comment by jbrew30** [comment]
I can second that. The Publishers are receiving emails but the editors are not receiving the email that the entry has been approved. 

Plugin version: 1.5.3
Craft version: 3.4.23

Setting
<img width="1134" alt="Screen Shot 2020-06-16 at 9 09 24 AM" src="https://user-images.githubusercontent.com/22452733/84778658-4b8faa00-afb1-11ea-841e-fa57f378cd71.png">

**Comment by engram-design** [comment]
I'll investigate shortly. Can you send through your workflow.log files? I've added some more logging to help diagnose the issue in 1.5.3

**Comment by engram-design** [comment]
Fixed in https://github.com/verbb/workflow/commit/bf1ff385219ff8e07c4e8dc6e702d9c8c9e30322 - will roll into the next release.

**Comment by jbrew30** [comment]
I'm not sure if you still need this, but here is the error that is getting logged. 

`$_POST = []
2020-06-17 08:32:44 [-][546][-][info][workflow] Preparing editor notification.
2020-06-17 08:32:44 [-][546][-][error][workflow] Failed to send editor notification to j******@gmail.com - “Calling unknown method: craft\mail\Mailer::setTo()” /Users/jbrewer/Documents/repos/goml-refresh/vendor/yiisoft/yii2/base/Component.php:300
2020-06-17 08:32:44 [-][546][-][info][application] $_GET = [
    'draftId' => '112'
]`

**Comment by engram-design** [comment]
Fixed in [1.5.4](https://github.com/verbb/workflow/releases/tag/1.5.4)
